openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -23,8 +24,21 @@ def process(self, context):
 
         # Get note status, by default uses the task status for the note
         # if it is not specified in the configuration
-        note_status = context.data["kitsu_task"]["task_status_id"]
-        if self.set_status_note:
+        kitsu_task = context.data["kitsu_task"]
+        shortname = kitsu_task["task_status"]["short_name"]
+        note_status = kitsu_task["task_status_id"]
+        if self.set_status_note and next(
+            (
+                False
+                for se in self.status_exceptions
+                if shortname == se["short_name"]
+                and se["equality"].startswith("e")
```

Comment:
`startswith` is slower than `==` because of the function overhead, you should change it.